### PR TITLE
RESTAPI - Add delay in `test_data_path`

### PR DESCRIPTION
**Pull request by bingwang-ms**
### Description of PR

Summary:
This PR is to fix test failure of ```test_data_path```.
```
        expected = [{"nexthop": "100.3.152.52", "ip_prefix": "192.168.20.4/32", "vnid": 7036001},
                    {"nexthop": "100.3.152.52", "ip_prefix": "101.0.20.5/32", "mac_address": "1c:34:da:72:b0:8a", "vnid": 7036001},
                    {"nexthop": "100.3.152.52", "ip_prefix": "100.0.20.4/32", "vnid": 7036001}]
        for route in expected:
>           pytest_assert(route in r.json())
E           Failed: <Failed instance>

```

Signed-off-by: bingwang <bingwang@microsoft.com>
### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
This PR is to fix test failure of ```test_data_path```.

#### How did you do it?
Delay 5 seconds before querying routes.

#### How did you verify/test it?
Verified on SN4600, test case can pass now.

#### Any platform specific information?
No.

#### Supported testbed topology if it's a new test case?
No.

### CODE REVIEW

File: tests/restapi/test_restapi.py
Comment (sumukhatv) [inline tests/restapi/test_restapi.py:110]: Why do we need this? We are not seeing any failure on Mellanox platform at the moment.

Comment (bingwang-ms) [reply tests/restapi/test_restapi.py:110]: The case is consistently failing on SN4600.

Comment (prsunny) [reply tests/restapi/test_restapi.py:110]: @sumukhatv , the get request here is taking 1.2 second. Seems like we need to investigate this:

```
Jul 15 02:03:10.442687 str-msn4600c-acs-02 INFO restapi#supervisord: restapi [ trace ] 2021/07/15 02:03:10 release server write lock
Jul 15 02:03:10.442687 str-msn4600c-acs-02 INFO restapi#supervisord: restapi [  info ] 2021/07/15 02:03:10 request: duration 1.237732764s
```
A previous SET request is just 2ms:
```
Jul 15 02:03:09.188731 str-msn4600c-acs-02 INFO restapi#supervisord: restapi [  info ] 2021/07/15 02:03:09 request: duration 2.778701ms
```

**Review by prsunny — APPROVED** [review]

**Comment by prsunny** [comment]
Please wait for Sumukha's approval before merge

**Review by sumukhatv — APPROVED** [review]

**Pull request merged**
